source/includes/kubernetes_extension/_k8_deployments.md

```diff
@@ -7,67 +7,280 @@
 ```shell
 curl -X GET \
    -H "MC-Api-Key: your_api_key" \
-   "https://cloudmc_endpoint/v1/services/k8s/an_environment/deployments"
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/deployments?cluster_id=a_cluster_id"
 ```
 
 > The above command returns a JSON structured like this:
 
 ```json
 {
-  "data": [
-    {
-      "id": "apm-server-1585600296-apm-server/monitoring",
-      "deploymentStatus": "Available",
-      "readyRatio": "1/1",
-      "metadata": {
-        "annotations": {
-          "deployment.kubernetes.io/revision": "1"
+    "data": [
```

Comment:
is all this data really necessary to display?

Reply:
I could change all the nested objects to be an empty object 

```
metadata: { ... },
spec: {...},
etc.
```

Reply:
I think that's a good call! Will reduce clutter + noise

Reply:
Done  363ce1f7606be98ce0bf0a53a32895438f13fd0f